CMakeLists.txt

```diff
@@ -549,6 +550,18 @@ if(AVIF_BUILD_TESTS)
     target_link_libraries(aviftest avif ${AVIF_PLATFORM_LIBRARIES})
     target_include_directories(aviftest PRIVATE apps/shared)
 
+    add_executable(avify4mtest
+        apps/shared/y4m.c
+        tests/avify4mtest.c
+    )
+    if(AVIF_LOCAL_LIBGAV1)
+        set_target_properties(avify4mtest PROPERTIES LINKER_LANGUAGE "CXX")
+    endif()
+    target_link_libraries(avify4mtest avif ${AVIF_PLATFORM_LIBRARIES})
+    target_include_directories(avify4mtest PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+    add_test(NAME avify4mtest
+        COMMAND ${CMAKE_BINARY_DIR}/avify4mtest)
```

Comment:
Why don't the other two tests (aviftest and avifyuv) need an add_test()?

Instead of this add_test(), can we add avify4mtest to the add_custom_target(avif_test_all ...) call at line 573?